*** ace_orig.js	Fri Nov 22 12:15:30 2019
--- ace.js	Fri Nov 22 12:13:27 2019
*************** if (typeof document !== "undefined") {
*** 1240,1246 ****
      div = null;
  }
  
! if (exports.HAS_CSS_TRANSFORMS) {
      exports.translate = function(element, tx, ty) {
          element.style.transform = "translate(" + Math.round(tx) + "px, " + Math.round(ty) +"px)";
      };
--- 1240,1246 ----
      div = null;
  }
  
! if (false && exports.HAS_CSS_TRANSFORMS) {
      exports.translate = function(element, tx, ty) {
          element.style.transform = "translate(" + Math.round(tx) + "px, " + Math.round(ty) +"px)";
      };
*************** var Marker = function(parentEl) {
*** 16193,16198 ****
--- 16193,16199 ----
          }
          x.style.cssText = css;
          x.className = className;
+         return x;
      };
  
      this.update = function(config) {
*************** var Marker = function(parentEl) {
*** 16308,16314 ****
          
          var radiusClass = (range.start.column ? 1 : 0) | (range.end.column ? 0 : 8);
  
!         this.elt(
              clazz + (radiusClass ? " ace_br" + radiusClass : ""),
              "height:"+ height+ "px;"+
              "right:0;"+
--- 16309,16315 ----
          
          var radiusClass = (range.start.column ? 1 : 0) | (range.end.column ? 0 : 8);
  
!         return this.elt(
              clazz + (radiusClass ? " ace_br" + radiusClass : ""),
              "height:"+ height+ "px;"+
              "right:0;"+
*************** var Marker = function(parentEl) {
*** 16325,16331 ****
          var top = this.$getTop(range.start.row, config);
          var left = this.$padding + range.start.column * config.characterWidth;
  
!         this.elt(
              clazz,
              "height:"+ height+ "px;"+
              "width:"+ width+ "px;"+
--- 16326,16332 ----
          var top = this.$getTop(range.start.row, config);
          var left = this.$padding + range.start.column * config.characterWidth;
  
!         return this.elt(
              clazz,
              "height:"+ height+ "px;"+
              "width:"+ width+ "px;"+
*************** var Marker = function(parentEl) {
*** 16354,16360 ****
          if (range.start.row != range.end.row)
              height += this.$getTop(range.end.row, config) - top;
  
!         this.elt(
              clazz,
              "height:"+ height+ "px;"+
              "top:"+ top+ "px;"+
--- 16355,16361 ----
          if (range.start.row != range.end.row)
              height += this.$getTop(range.end.row, config) - top;
  
!         return this.elt(
              clazz,
              "height:"+ height+ "px;"+
              "top:"+ top+ "px;"+
*************** var Marker = function(parentEl) {
*** 16366,16372 ****
          var top = this.$getTop(range.start.row, config);
          var height = config.lineHeight;
  
!         this.elt(
              clazz,
              "height:"+ height+ "px;"+
              "top:"+ top+ "px;"+
--- 16367,16373 ----
          var top = this.$getTop(range.start.row, config);
          var height = config.lineHeight;
  
!         return this.elt(
              clazz,
              "height:"+ height+ "px;"+
              "top:"+ top+ "px;"+
